.readthedocs.yml

```diff
@@ -10,6 +10,13 @@ build:
   os: ubuntu-22.04
   tools:
     python: "3.11"
+
+  # Unshallow the git clone to allow vcs versioning to pick
+  # up tags properly
+  jobs:
+    post_checkout:
+      - git fetch --unshallow || true
```

Comment:
Do we really need the `|| true` there? I know it's what the upstream doc says, but I don't think it would ever fail for us

Reply:
Yes, the `|| true` part is just there to ensure that the command succeeds even if the repository isn't shallow to begin with. But as you mentioned, it shouldn't fail now. I'll drop it.